### PR TITLE
add: `tre`

### PR DESCRIPTION
**Pull request by DiegoMagdaleno**

### CODE REVIEW

**Comment by Henryws** [comment]
Rename the file to `packages/tre-git/tre-git.pacscript`

**Comment by DiegoMagdaleno** [comment]
Done

File: packages/tre-git/tre-git.pacscript [hunk 1]
```diff
@@ -0,0 +1,24 @@
+name="tre-git"
+pkgname="tre"
+gives="tre"
+build_depends="cargo rustc"
+description="Tree command, improved."
```
Comment (Henryws) [inline packages/tre-git/tre-git.pacscript:5]:
```suggestion
description="Tree command, improved"
```

File: packages/tre-git/tre-git.pacscript [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+url="https://github.com/dduan/tre"
```
Comment (Henryws) [inline packages/tre-git/tre-git.pacscript:6]:
```suggestion
url="https://github.com/dduan/tre.git"
```

File: packages/tre-git/tre-git.pacscript [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+maintainer="Diego Magdaleno <diegomagdaleno@pm.me>"
+pkgdir="${STOWDIR}/${name}"
```
Comment (Henryws) [inline packages/tre-git/tre-git.pacscript:8]:
```suggestion
```

File: packages/tre-git/tre-git.pacscript [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+pkgver() {
+	git ls-remote "${url}" main | cut -f1 | cut -c1-8
+}
+version="$(pkgver)"
+
+prepare() {
+	true
+}
```
Comment (Henryws) [inline packages/tre-git/tre-git.pacscript:16]:
```suggestion
```

File: packages/tre-git/tre-git.pacscript [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+
+build() {
+	cargo build -j$(nproc) --release
```
Comment (Henryws) [inline packages/tre-git/tre-git.pacscript:19]:
```suggestion
	cargo build -j$(nproc) --release --locked
```

**Review by Henryws — CHANGES_REQUESTED** [review]

File: packages/tre-git/tre-git.pacscript [hunk 6]
```diff
@@ -0,0 +1,19 @@
+name="tre-git"
+pkgname="tre"
+gives="tre"
+build_depends="cargo rustc"
+description="Tree command, improved"
+url="https://github.com/dduan/tre.git"
+maintainer="Diego Magdaleno <diegomagdaleno@pm.me>"
```
Comment (Henryws) [inline packages/tre-git/tre-git.pacscript:7]:
```suggestion
maintainer="Diego Magdaleno <diegomagdaleno@pm.me>"
incompatible=("debian:bullseye")
```

**Review by Henryws — CHANGES_REQUESTED** [review]

**Review by Henryws — APPROVED** [review]

**Pull request merged**